pandas/core/frame.py

```diff
@@ -4621,8 +4624,13 @@ def drop_duplicates(
         if inplace:
             (inds,) = (-duplicated)._ndarray_values.nonzero()
             new_data = self._data.take(inds)
+            if ignore_index:
+                new_data.axes[1] = ibase.default_index(len(inds))
```

Comment:
I would add a newline here and above the `if` statement for better readability.

Reply:
thanks! newline added above `if`